tests/phpunit/includes/object-cache.php

```diff
@@ -482,6 +487,7 @@ function wp_cache_get_multi_by_key( $server_key, $keys, $groups = '', &$cas_toke
  */
 function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 	global $wp_object_cache;
+	$wp_object_cache->getMulti( $keys, $group );
```

Comment:
```suggestion

	// Prime multiple keys in a single Memcached call.
	$wp_object_cache->getMulti( $keys, $group );

```

Reply:
It was not clear to me at a glance what this line does, as it does not return the result.

I see now that `getMulti()` retrieves multiple values in a single Memcached request and saves them in the `$cache` property, making subsequent `get()` calls faster.

I think adding a comment to clarify this a bit would be helpful.